TF_Service_and_Tooling_Unity/Assets/TouchFree/Service/Scripts/Interactions/InteractionModules/TouchPlanePushInteraction.cs

```diff
@@ -18,6 +18,10 @@ public class TouchPlanePushInteraction : InteractionModule
 
         private Vector2 downPos;
 
+        // Used to ignore hands that initialise while past the touchPlane.
+        // Particularly for those that are cancelled by InteractionZones
+        bool handWasCancelled = true;
```

Comment:
Might not need changing, but is it only the case that this is true when a hand is cancelled? 
From looking through the code it appears this would also be true when a hand first appears, regardless of whether the interaction zone has cancelled it or if it has just been detected for the first time. I might be missing something though.

Reply:
Yes so this also is set to true every time tracking is lost, so it may be that we want to rename the variable. I couldn't think of anything more suitable but if you can, we can change it :)

Reply:
`handWasNull` would be the simplest option as it says exactly what we know about the hand, but it doesn't capture the part about it only clearing when your hand is outside the touch plane.
`handNotReady` helps imply that there is a condition attached, but is confusing to be testing if `!handNotReady` as it's a double negative. 
`handReady` would read better, but would mean inverting some of your logic for it to make sense. ie `if (handReady)` in place of `if (!handWasCancelled)` and making it be false when the hand is null

Reply:
I will implement handReady as that makes the most sense to me too :)